pkg/analysis_server/lib/src/services/correction/fix_internal.dart

```diff
@@ -1784,6 +1787,16 @@ class FixProcessor {
     _addFix(DartFixKind.ADD_NE_NULL, []);
   }
 
+  void _addFix_removeAwait() {
+    AwaitExpression awaitExpression = node;
+    if (awaitExpression != null) {
+      final awaitToken = awaitExpression.awaitKeyword;
+      _addRemoveEdit(
+          rf.rangeStartEnd(awaitToken.offset, awaitToken.next.offset));
+      _addFix(DartFixKind.REMOVE_AWAIT, []);
+    }
```

Comment:
This works, but has the disadvantage that if the node is ever not an await expression we'll get a cast exception at runtime. That's why I was recommending
```
if (node is AwaitExpression) {
  final awaitToken = node.awaitKeyword;
  ...
}
```
Worst case with the code above is that we silently don't produce a fix.